src/main/webapp/cdn/project.js

```diff
@@ -19,7 +19,7 @@ var projectTypes = {
 var simplemde = null;
 
 $(document).ready(function () {
-    simplemde = new SimpleMDE({element: document.getElementById("project-form-description")});
+    simplemde = new SimpleMDE({element: document.getElementById("project-form-description"), hideIcons: ["link"], spellChecker: false});
```

Comment:
Removes spell checking and the button that allows users to put URLs in their project descriptions.